### ISSUE TITLE
load_timing: domReady don't work with async 

### ISSUE THREAD

**Issue by ArekZw** [style-tools/async#1]
Reproduce:

```
<script async src="async-iife.js" data-c='
[
    [
	    "loaded.css", {
	        "4": "not_loaded.css",
	        "load_timing": "domReady"
	    }
    ]
]'>
</script>
```

**Comment by ArekZw** [comment]
Same for load_timing: inview - without async it works.

**Comment by optimalisatie** [comment]
Hi @ArekZw,

Thank you for reporting the first bug!

There was an issue with the `domReady` check. It has been fixed in version `1.0.6`.

Test: https://style.tools/tests/data-c-async.html

We have also added a JSON compressor to [async-iife](https://github.com/style-tools/async-iife/) which is available online on https://style.tools/iife/

The config in your example could be further compressed to the following:

```json
["loaded.css",{"4":"not_loaded.css","48":54}]
```

https://github.com/style-tools/async-iife/releases/tag/1.0.7
